satellite/templates/VM_info.html

```diff
@@ -37,12 +37,10 @@ <h2 class="card-pf-title">
                     </h2>
                 </div>
                 <div class="card-pf-body" id="vm_packages">
-                    <table class="table table-striped table-bordered table-hover" id="myPackages">
+                    <table class="table table-striped table-bordered table-hover">
                         <thead>
                         <tr>
-                            <th>Package Name
-                            <input style="float:right" type="text" id="myInput" onkeyup="myFunction()" placeholder="Search for packages...">
-                            </th>
```

Comment:
@CURIOUSER005 This textfield is needed to search packages. Where will the user enter the name of the package now?